react-data-converter/.gitignore

```diff
@@ -0,0 +1,13 @@
+# Client Ignores
```

Comment:
I was wondering if we want to move this into a sub-directory inside API-Zoo? Thoughts?